digitaltwin_client/src/dt_interface_client.c

```diff
@@ -1200,27 +1200,22 @@ static void ProcessPropertiesForTwin(DT_INTERFACE_CLIENT* dtInterfaceClient, JSO
     JSON_Object* dtInterfaceDesiredJson;
     JSON_Object* dtInterfaceReportedJson;
 
-    STRING_HANDLE desiredPathForInterface = NULL;
-    STRING_HANDLE reportedPathForInterface = NULL;
+    STRING_HANDLE pathForInterface = NULL;
 
     // If there are property updates for this interface - which there may or may not be - the will in the passed in JSON 
```

Comment:
>the [](start = 90, length = 3)

Old typo but please fix while you're in here

"the will in the passed" should be
"they will be in the passed"